shared/js/background/helpers/arguments-object.js

```diff
@@ -26,10 +25,17 @@ export function getArgumentsObject (tabId, sender, documentUrl, sessionKey) {
         site.enabledFeatures = site.enabledFeatures.filter(feature => aboutBlankEnabled.includes(feature))
     }
 
+    site.enabledFeatures = site.enabledFeatures.filter((feature) => {
+        // Prune out the tracker allowlist feature setting as it's not needed and is large
+        if (feature === 'trackerAllowlist') return false
```

Comment:
Moving this has the same effect as pruning below, merging it makes the below code easier to follow if feature pruning is only here.